notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>But a Series still has a name ... </p><p>Is that something to mention? In the example above we also give it a name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>